tests/Operation/ModifyCollectionTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Exception\InvalidArgumentException;
+use MongoDB\Operation\ModifyCollection;
+
+class ModifyCollectionTest extends TestCase
+{
```

Comment:
If you end up adding a non-empty check for `$collectionOptions`, we can have another test here to demonstrate that `[]` throws an exception.